CMakeLists.txt

```diff
@@ -46,6 +46,8 @@ project(faiss
   LANGUAGES ${FAISS_LANGUAGES})
 include(GNUInstallDirs)
 
+set(CMAKE_INSTALL_PREFIX "$ENV{CONDA_PREFIX}")
```

Comment:
Not sure we want to keep this as Faiss is not necessarily compiled in a conda env.

Reply:
Ah, sorry. This was just for development purposes - I will remove it.